### ISSUE TITLE
Issues on Windows platform

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/VirES-Python-Client#29]
Issues observed on v0.3.0.

Trying to convert ReturnedData to DataFrame the following exception is raised:

```python
Exception: Bad or empty cdf.
```
Trying to convert ReturnedData to xarray the following exception is raised:

```python
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\LUCAMA~1\\AppData\\Local\\Temp\\tmpxhsjw4u_'
```

Maybe the issue is due to the path containing a blank space: my home directory is

```
C:\Users\Luca Mariani
```

In the attached [test_viresclient.zip](https://github.com/ESA-VirES/VirES-Python-Client/files/3181889/test_viresclient.zip) you can find the Notebook with the test execution.

**Comment by smithara** [comment]
See https://github.com/ESA-VirES/VirES-Python-Client/issues/1

Sorry I didn't come up with a better way of handling it and Windows compatibility was a low priority (for me)

**Comment by smithara** [comment]
I think this will be fixed now in the `windows_support` branch. If you have time you can try it out:

`pip install git+https://github.com/ESA-VirES/VirES-Python-Client@windows_support`

I will check it over a bit more but will make a release soon with a number of other changes.

**Comment by smithara** [comment]
I think it's working okay now (released viresclient v0.4.0). Let me know if you run into problems again.
